recipes/libastra/meta.yaml

```
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "libastra" %}
+{% set version = "1.9.9" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tomopy/astra-toolbox.git
+  git_tag: master
+
+build:
+  number: {{ build }}
+  string: "{{ "cuda" + cuda_compiler_version|string }}h{{ PKG_HASH }}_{{ build }}"
+  skip: true  # [osx or win or cuda_compiler_version == "None"]
+  merge_build_host: True  # [win]
+  script:
+    cmake
+    -G Ninja
+    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX
+    -DCMAKE_INSTALL_LIBDIR=lib
+    ${CMAKE_ARGS} ${SRC_DIR} && ninja install
+
+requirements:
+  build:
+    - {{ compiler('cuda') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - boost
```

You probably don't need Boost.Python? Then the following should be sufficient.
```suggestion
    - boost-cpp
```